include/oneapi/internal/math/blas_executor.h

```diff
@@ -24,13 +24,7 @@
 //--
 */
 
-#ifdef ONEAPI_DAAL_USE_MKL_GPU_FUNC
```

Comment:
Why did you remove this condition? MKL is expected to work only with clang. I would replace it with check for Intel DPC++ compiler not only for clang